bootstrap/vars/config.sample.yaml

```diff
@@ -63,8 +69,11 @@ bootstrap_cluster_cidr: 10.42.0.0/16
 # (Advanced) For ipv6 use format 10.43.0.0/16,fd78:c889:47fb:e0::/112
 bootstrap_service_cidr: 10.43.0.0/16
 
-# (Advanced) Path for democratic-csi local-hostpath to use for storage on ALL nodes
-bootstrap_local_path_provisioner_path: /var/lib/rancher/k3s/local-hostpath
+# (Advanced) Path for openebs openebs-hostpath to use for storage on ALL nodes
+# for k3s distribution
+#bootstrap_local_storage_path: /var/lib/rancher/k3s/local-hostpath
+# for k0s distribution
+#bootstrap_local_storage_path: /var/openebs/local
```

Comment:
```suggestion
# (Advanced) Path for openebs openebs-hostpath to use for storage on ALL nodes
bootstrap_local_storage_path: /var/openebs/local
```

We can now default this.

Reply:
Right, I added it to the required values validator and removed the default value in the template as well